ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__ppc64le-nightly.yaml

```diff
@@ -0,0 +1,110 @@
+base_images:
+  ansible:
+    name: "4.17"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.17"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  hypershift-operator:
+    name: "4.17"
+    namespace: ocp
+    tag: hypershift-operator
+  openstack-installer:
+    name: "4.17"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.17"
+  tools:
+    name: "4.17"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.17"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    candidate:
+      architecture: multi
```

Comment:
@aishwaryabk per my understanding

```yaml
  latest:
    candidate:
      architecture: multi
      product: ocp
      stream: nightly
      version: "4.17"
```
to
```yaml
  latest:
    candidate:
      architecture: amd64
      product: ocp
      stream: nightly
      version: "4.17"
  multi-latest:
    candidate:
      architecture: multi
      product: ocp
      stream: nightly
      version: "4.17"
```
if you need multi release
or
```yaml
  latest:
    candidate:
      architecture: amd64
      product: ocp
      stream: nightly
      version: "4.17"
```
if you do no need multi release.

the key is to keep latest as amd for amd oc on prow agent.

@liangxia please correct me if I am wrong.

Reply:
@kuiwang02 The job runs on IBM power (ppc64le). So, the multi-release will be fine and will not require an amd64 release.

Reply:
@aishwaryabk for ppc64le cluster, it use ppc64le-latest.
the latest
```yaml
  latest:
    candidate:
      architecture: amd64
      product: ocp
      stream: nightly
      version: "4.17"
```
is used for oc inject. the prow build farm is amd64, and prow agent is running on it. so, it need amd64 for oc.
so, have to define `latest` because oc inject need it, and have to be amd64..

for multi-latest, you could keep it if you need it, but it is not used for oc inject.

Reply:
anyway, you could keep your definiton and  `/pj-rehearse` your job to see what happen.

I do not find you ever pj-rehearse the job.

Reply:
and more, I find when arm related prow jobs (ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__arm64-nightly.yaml) are added, the 
```yaml
releases:
  arm64-latest:
    candidate:
      architecture: arm64
      product: ocp
      stream: nightly
      version: "4.17"
  latest:
    candidate:
      architecture: amd64
      product: ocp
      stream: nightly
      version: "4.17"
  target:
    candidate:
      architecture: multi
      product: ocp
      stream: nightly
      version: "4.17"
```
latest is for amd64. I do not know why, but at least latest is amd64.

and multi related prow job (ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml), same latest for amd64
```yaml
eleases:
  latest:
    candidate:
      architecture: amd64
      product: ocp
      stream: nightly
      version: "4.17"
  multi-latest:
    candidate:
      architecture: multi
      product: ocp
      stream: nightly
      version: "4.17"
```

it is not me to create these jobs, but per my understanding, it is latest used for oc injection.

Reply:
Okay, I can add this amd64 part if required. I was using the [4.16 file](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__ppc64le-nightly.yaml) for reference which did not have this part.

Reply:
@aishwaryabk I checked 4.16 job build https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.16-ppc64le-nightly-powervs-ipi-f7/1811188919883010048/artifacts/powervs-ipi-f7/openshift-extended-test/build-log.txt, and do not find oc fail. so, it should work.

So, you could keep it. please ignore my comment.
I am ok for this PR.

by the way, once you find the oc fails with error `exec format error`, you could change it.